echopype/convert/set_groups_ad2cp.py

```diff
@@ -346,32 +367,23 @@ def set_vendor(self) -> xr.Dataset:
 
         return set_encodings(ds)
 
-    def set_beam_complex(self) -> xr.Dataset:
+    def set_sonar(self) -> xr.Dataset:
         ds = xr.Dataset(
-            data_vars={
-                "echosounder_raw_samples_r": self.ds.get("echosounder_raw_samples_r"),
-                "echosounder_raw_samples_i": self.ds.get("echosounder_raw_samples_i"),
-                "echosounder_raw_transmit_samples_r": self.ds.get(
-                    "echosounder_raw_transmit_samples_r"
-                ),
-                "echosounder_raw_transmit_samples_i": self.ds.get(
-                    "echosounder_raw_transmit_samples_i"
-                ),
-                "echosounder_raw_beam": self.ds.get("echosounder_raw_beam"),
-                "echosounder_raw_echogram": self.ds.get("echosounder_raw_echogram"),
-            },
-            coords={
-                "ping_time": self.ds.get("ping_time"),
-                "ping_time_echosounder_raw": self.ds.get("ping_time_echosounder_raw"),
-                "ping_time_echosounder_raw_transmit": self.ds.get(
-                    "ping_time_echosounder_raw_transmit"
-                ),
-                "sample": self.ds.get("sample"),
-                "sample_transmit": self.ds.get("sample_transmit"),
-            },
-            attrs={"pulse_compressed": self.pulse_compressed},
+            attrs={
+                "sonar_manufacturer": "Nortek",
+                "sonar_model": "AD2CP",
+                "sonar_serial_number": "",
+                "sonar_software_name": "",
+                "sonar_software_version": "",
+                "sonar_firmware_version": "",
+                "sonar_type": "acoustic Doppler current profiler",
```

Comment:
Tiny comment: it seems odd to only use upper case for Doppler, though I see why you'd do that. I couldn't find a specific vocabulary term for ADCP sonar_type in the ICES vocabularies. How about "Acoustic Doppler Current Profiler (ADCP)"?

Reply:
Please reverse this change -- it should actually be "acoustic Doppler current profiler (ADCP)". Doppler is capitalized because it is a name. This is what wikipedia uses also.